### DISCUSSION TITLE
projection of the structure contours on the beam's eye view?

### DISCUSSION THREAD

**Discussion by gourav3017** [e0404/matRad#573] [Q&A]
Can we get any  data for the projection of structure contours (mask) on beam's eye view using matRad?

**Comment by wahln** [comment]
Hi @gourav3017, 
sorry for the delayed answer, for some reason it's difficult to get the notifications running for GitHub Discussions.
There's no native implementation for what you are asking, but it should be able to get those. During dose calculation, the patient is rotated into beams-eye-view (in matRad_calcDoseInitBeam). With a little bit of coding, the projections of the contours should be extractable from the rotated VOI masks.
Let use know if you are still interested in this and need help.
